documentation/docs/steps/cloudFoundryDeploy.md

```diff
@@ -2,6 +2,29 @@
 
 ## ${docGenDescription}
 
+### Additional Hints
```

Comment:
This new headline is required otherwise the text below would be stripped-away by the docu generator.